tests/system/feature_store/test_spark_engine.py

```diff
@@ -48,26 +48,29 @@
 from tests.system.feature_store.expected_stats import expected_stats
 
 
-def read_and_assert(csv_path_spark, csv_path_storey):
+def read_and_assert(out_path_spark, out_path_storey):
     read_back_df_spark = None
     file_system = fsspec.filesystem("v3io")
-    for file_entry in file_system.ls(csv_path_spark):
+    for file_entry in file_system.ls(out_path_spark):
         filepath = file_entry["name"]
         if not filepath.endswith("/_SUCCESS"):
-            read_back_df_spark = pd.read_csv(f"v3io://{filepath}")
+            read_back_df_spark = pd.read_parquet(f"v3io://{filepath}")
```

Comment:
All the steps tests such as `test_ingest_with_steps_drop_features` using this function and pass to it csv path, I think you can change all the target to be parquet or to check here witch file path you get and use the right function : `read_csv` or `read_parquet`.

And also why did you change in the `DateExtractor` test the target to be parquet?

Reply:
The question is why do they use CSV? The reason the test does not expose [ML-3414](https://jira.iguazeng.com/browse/ML-3414) is because CSV loses the type information.

I'll bring back the CSV implementation for exixting tests, but they should be migrated off of it if there is no good reason for them to be using CSV (such as if the purpose of the test is to test support for CSV).

Reply:
Thank, good to know!

About the Implementation I think it is better to change all to parquet or at least not to write 2 equally functions... but it can be change in another pr I guess.